src/theme/mui.js

```diff
@@ -162,41 +166,81 @@ const theme = createMuiTheme({
     },
     MuiInput: {
       root: {
-        fontFamily: secondaryFontFamily,
+        backgroundColor: '#F0EFEE',
```

Comment:
we have a var with this color `secondaryBackground` in `/src/theme/variables.js`

Reply:
fixed at https://github.com/gnosis/safe-react/pull/577/commits/9b4b609a36e8e74f48b08edf9fc2fff3c25cca5b, thanks!